src/System.Net.Sockets/tests/FunctionalTests/SendFile.cs

```diff
@@ -101,7 +101,6 @@ public void NotConnected_ThrowsException()
         public void SendFile_Synchronous(IPAddress listenAt, bool sendPreAndPostBuffers, int bytesToSend)
         {
             const int ListenBacklog = 1;
-            const int LingerTime = 10;
```

Comment:
I have thought about that problem. And yes, I want to propose changing all our tests to use Socket.Shutdown() before dispose().

>There are lots of other instances of this pattern in the same file

There are two instances in this file. I have changed one, the other one is in SendFile_APM(). I can submit a new PR for that.

>do we have other tests validating that LingerOptions work?

No for this test file. I will look into other tests validating LingerOptions inside the whole project scope.